native/cocos/bindings/auto/jsb_assets_auto.cpp

```diff
@@ -15701,17 +15701,14 @@ static bool js_assets_BuiltinResMgr_initBuiltinRes(se::State& s) // NOLINT(reada
     const auto& args = s.args();
     size_t argc = args.size();
     CC_UNUSED bool ok = true;
-    if (argc == 1) {
-        HolderType<cc::gfx::Device*, false> arg0 = {};
-        ok &= sevalue_to_native(args[0], &arg0, s.thisObject());
-        SE_PRECONDITION2(ok, false, "js_assets_BuiltinResMgr_initBuiltinRes : Error processing arguments");
-        bool result = cobj->initBuiltinRes(arg0.value());
+    if (argc == 0) {
+        bool result = cobj->initBuiltinRes();
```

Comment:
Don't need to commit code in bindings/auto folder, they will be committed by cocos-robot.

Reply:
reverted. Maybe those files should be add to .gitignore?